### ISSUE TITLE
Add FAQ document

### ISSUE THREAD

**Issue by Exadra37** [exadra37-versioning/explicit-versioning#5]
While Explicit Versioning is gaining traction, some questions will popup overtime and this questions may be already documented, but normally we Humans are good in overlook them when reading the documentation.

This can happens because:

* we are reading in a rush.
* not reading with attention.
* reading when tired.
* we are being simple lazy and instead of reading we just open a new issue and wait for somebody else to do the work for us.

As a Moderator in a forum I see all them happen a lot, specially the last one.

So the FAQ will be a place to put the Question and Reply for the most pertinent questions we will find overtime.

Anyway when a question popups and the document is referring to is not clear enough, than we must amend the document instead.

This FAQ document will be separated from the README holding the Specification.

**Comment by Exadra37** [comment]
@sapioit @colomet @bmitch

Any suggestions about questions to go into the FAQ?

**Comment by bmitch** [comment]
@Exadra37 I could write up some questions if you want to answer them?

**Comment by Exadra37** [comment]
@bmitch put here your questions than we will see if they are a good fit for the FAQ or the documents need to be updated to clarify that questions ;)

**Comment by bmitch** [comment]
Just off the top of my head:

* What is explicit versioning?
* How does it differ from other versioning methods?
* What advantages does explicit versioning provide?
* Is it compatible with composer (or other package management software).
* Who is behind explicit versioning?
* When should I use explicit versioning?
* Why should I use  explicit versioning?

**Comment by Exadra37** [comment]
@bmitch nice set of FAQ questions... thanks :)

**Comment by sapioit** [comment]
Also:

* What explicit versioning is not?
* What disadvantages does explicit versioning have?
* When should I not use explicit versioning?
* Why should I not use explicit versioning?

Just to make things more... explicit. (I know, I know, bad pun...)

**Comment by Exadra37** [comment]
@sapioit 

Since when Explicit Versioning has disadavantages :roll_eyes:

**Comment by sapioit** [comment]
Since there are people who prefer a different versioning system. That would be the only disadvantage I can think of...

**Comment by Exadra37** [comment]
@sapioit 

I was making a joke ;)

Disadvantages:

* you need to write 1 more digit and a dot.
* enforces thinking more, once now is clear when a Incompatible Release is done.
* encourages use of strong suites of tests to ensure that breaking changes will not happen without being intentional.
